proto/cometbft/p2p/v1/pex.proto

```
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package cometbft.p2p.v1;
+
+option go_package = "github.com/cometbft/cometbft/api/cometbft/p2p/v1";
+
+import "cometbft/p2p/v1/types.proto";
```

Buf (1.47.2)

import "cometbft/p2p/v1/types.proto": file does not exist

(COMPILE)